src/main/resources/application.yml

```diff
@@ -99,6 +99,7 @@ jira:
     X: Y # Map Checkmarx severity : to JIRA Priority
   open-transition: Reopen Issue
   close-transition: Close Issue
+# project-key-script: D:\\tmp\JiraProjectKey.groovy
```

Comment:
Will it work with a single slash?
```suggestion
# project-key-script: D:\tmp\JiraProjectKey.groovy
```

Reply:
it's only an example taken from comment script (different file name)

Reply:
changed to:
 # project-key-script: \tmp\JiraProjectKey.groovy

Reply:
I removed the path example